client-web/src/functional-e2e/test-plan-applications.md

```diff
@@ -0,0 +1,897 @@
+# Space Applications Management - Comprehensive Test Plan
+
+## Application Overview
+
+The Alkemio platform provides a multi-level space hierarchy (Space → Subspace → Subsubspace) with application-based membership. The application workflow allows non-members to apply for membership, complete questionnaires, and have their applications reviewed by space administrators with inherited permissions.
+
+**Key Features:**
+
+- **3-Level Space Hierarchy**: Level 0 (Space), Level 1 (Subspace), Level 2 (Subsubspace)
+- **Privacy Indicators**: Lock icons indicate private spaces
+- **Application Workflow**: Apply → Answer Questions → Review → Approve/Reject/Delete
+- **Inherited Admin Permissions**: Admin rights cascade down the hierarchy
+- **Notification System**: Real-time notifications for applicants and reviewers
+- **Application Management**: View questionnaires, approve, reject, or delete applications
+
+**User Roles:**
+
+- **Applicant**: `${TestUser.NON_SPACE_MEMBER}@alkem.io` - Non-member applying to spaces
+- **Level 0 Admin**: `${TestUser.SPACE_ADMIN}@alkem.io` - Admin of all 3 levels
+- **Level 1 Admin**: `${TestUser.SUBSPACE_ADMIN}@alkem.io` - Admin of Level 1 & 2
+- **Level 2 Admin**: `${TestUser.SUBSUBSPACE_ADMIN}@alkem.io` - Admin of Level 2 only
+
+**Navigation Elements:**
+
+- **Settings Access**: Cog icon in subheader navigation menu
+- **Notifications**: Bell icon in upper right corner
+- **Community Tab**: Subnavigation within settings
+
+---
+
+## Test Scenarios
+
+### 1. Space Discovery and Privacy Indicators
+
+**Seed:** `./client-web/src/functional-e2e/seed-applications.spec.ts`
+
+#### 1.1 View Private Space Hierarchy as Non-Member
+
+**Steps:**
+
+1. Sign in as `${TestUser.NON_SPACE_MEMBER}@alkem.io`
+2. Navigate to the main spaces listing page
+3. Locate the 3-level space tree structure
+4. Observe privacy indicators for each level
+
+**Expected Results:**
+
+- Level 0 (Space) displays with lock icon indicating private status
+- Level 1 (Subspace) displays with lock icon indicating private status
+- Level 2 (Subsubspace) displays with lock icon indicating private status
+- All three levels show "Apply" or "Join" button available
+- Non-member cannot directly access space content
+
+#### 1.2 Verify Space Hierarchy Structure
```

Comment:
This one is obsolete for private space if user is not a member, which is the case in this scenario.

Could be verified for public journeys